content-repo/extra-docs/articles/tim-edl.md

```diff
@@ -1,164 +1,190 @@
 
-3. Define a feed-triggered job that runs every time indicators are ingested by the feed.
+3. [Customize the playbooks](#customize-your-playbooks): These playbooks process the indicators and determine which are legitimate.
 
-4. Review the indicators and determine with which tags each indicator should be tagged.
+4. [Define a feed-triggered job](#define-a-feed-triggered-job): Run a job every time indicators are ingested by the feed. 
 
-5. Customize a job that is time-based to push the indicators to the SIEM.
+5. [Review the indicators to send to the SIEM](#customize-the-add-all-indicator-types-to-siem-playbook): Determines which indicators are sent to the SIEM.
 
-This article does not cover every possible scenario in this flow. It provides a real-life scenario that touches on important points from which you can extrapolate to other scenarios.
+6. [Customize a job to push the indicators to the SIEM](#define-a-job-to-push-the-indicators-to-the-siem): Run a time-based job to push the indicators to the SIEM.
 
-## Configure the Threat Intel Feed
 
-Cortex XSOAR comes with the several TIM feeds out of the box. In this flow, we will define the Bambenek Consulting feed.
 
-1. Navigate to **Settings > Integrations > Servers and Services** and search for Bambenek.
+## Configure the Threat Intel Feed
 
-2. Click **Add instance**.
+You can install several TIM feeds in the Marketplace. In this example, we will use the **Bambenek Consulting feed**.
 
-    1. Under **Services**, select from where you want to pull the data. For example, *All Indicator Feed*.
+1. Go to **Marketplace** and download **Bambenek Consulting feed**.
+2. Go to **Settings > INTEGRATIONS > Instances** and search for **Bambenek Consulting Feed**.
 
-    2. Determine the default reputation that you want indicators from this feed to receive. 
+3. Click **Add instance**.
 
-    3. You can change the **Source Reliability**. By default, Cortex XSOAR defines this feed as C - fairly reliable.
+    1. Select **Fetches indicators**. 
+    2. Determine the default reputation that you want indicators from this feed to receive. By default this is set to **Malicious**.
 
-    4. Determine when the indicator expires and how often to fetch indicators from the feed.
+    3. Change the **Source Reliability**, if required. By default, Cortex XSOAR defines this feed as **B - Usually reliable**.
+    4. In the **Services** field, select from where you want to pull the data. For example, **C2 All Indicator Feed**.
 
-    ![Threat Feed](../../../docs/doc_imgs/tutorials/tutorial-playbook-TIM-EDL/tutorial_playbook_tim_feed.png "Threat Feed")
+    5. Determine when the indicator expires and how often to fetch indicators from the feed.
 
-3. Click **Done**.
+   4. Click **Test**.
+   5. Click **Save & exit**.
+   The Indicators are now being ingested into Cortex XSOAR.
 
-## Create List of Indicators not to Process
+## Create a List of Indicators Not to Process
```

Comment:
Don't we have to add a list so that some indicators are not added to the block list? I think this is what is being referred to here https://xsoar.pan.dev/docs/reference/playbooks/tim---indicator-auto-processing